### DISCUSSION TITLE
Bootstrap failed installation

### DISCUSSION THREAD

**Discussion by EmanueleMiz** [okd-project/okd#706] [General]
**Describe the bug**
I can' t finish installing bootstrap machine where i am wrong  ?

**Version**
OKD version release: 4.7.0-0.okd-2021-04-11-124433
Fedora version : 33.20210217.3.0
UPI install method to vSphere

**Log bundle**
Apr 26 14:04:54 bootstrap release-image-download.sh[1856]: Error: 2 errors occurred:
Apr 26 14:04:54 bootstrap release-image-download.sh[1856]:         * error copying to host: error during bulk transfer for copier.request{Request:"PUT", Root:"/", preservedRoot:"/run/mco-machine-os-content/os-content-104617335", rootPrefix:"/run/mco-machine-os-content/os-content-104617335", Directory:"/", preservedDirectory:"/run/mco-machine-os-content/os-content-104617335", Globs:[]string{}, preservedGlobs:[]string{}, StatOptions:copier.StatOptions{CheckForArchives:false, Excludes:[]string(nil)}, GetOptions:copier.GetOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), Excludes:[]string(nil), ExpandArchives:false, ChownDirs:(*idtools.IDPair)(nil), ChmodDirs:(*os.FileMode)(nil), ChownFiles:(*idtools.IDPair)(nil), ChmodFiles:(*os.FileMode)(nil), StripSetuidBit:false, StripSetgidBit:false, StripStickyBit:false, StripXattrs:false, KeepDirectoryNames:false, Rename:map[string]string(nil), NoDerefSymlinks:false, IgnoreUnreadable:false}, PutOptions:copier.PutOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), DefaultDirOwner:(*idtools.IDPair)(nil), DefaultDirMode:(*os.FileMode)(nil), ChownDirs:(*idtools.IDPair)(0xc0004994c0), ChmodDirs:(*os.FileMode)(nil), ChownFiles:(*idtools.IDPair)(0xc0004994d0), ChmodFiles:(*os.FileMode)(nil), StripXattrs:false, IgnoreXattrErrors:false, IgnoreDevices:true, NoOverwriteDirNonDir:false, Rename:map[string]string(nil)}, MkdirOptions:copier.MkdirOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), ChownNew:(*idtools.IDPair)(nil), ChmodNew:(*os.FileMode)(nil)}}: copier: put: error setting extended attributes on "/extensions/okd/perl-Errno-1.30-468.fc33_1.30-469.fc33.x86_64.drpm": error setting value of extended attribute "user.Zif.MdChecksum[1617881158]" on "/extensions/okd/perl-Errno-1.30-468.fc33_1.30-469.fc33.x86_64.drpm": operation not supported
Apr 26 14:04:54 bootstrap release-image-download.sh[1856]:         * error copying from container: error during bulk transfer for copier.request{Request:"GET", Root:"/", preservedRoot:"/var/lib/containers/storage/overlay/22179495c990ceb5ecf49b26e1eddb6b7fd92c648a3ef9ba831688da4447c628/merged", rootPrefix:"/var/lib/containers/storage/overlay/22179495c990ceb5ecf49b26e1eddb6b7fd92c648a3ef9ba831688da4447c628/merged", Directory:"/", preservedDirectory:"/var/lib/containers/storage/overlay/22179495c990ceb5ecf49b26e1eddb6b7fd92c648a3ef9ba831688da4447c628/merged", Globs:[]string{"/"}, preservedGlobs:[]string{"/var/lib/containers/storage/overlay/22179495c990ceb5ecf49b26e1eddb6b7fd92c648a3ef9ba831688da4447c628/merged/."}, StatOptions:copier.StatOptions{CheckForArchives:false, Excludes:[]string(nil)}, GetOptions:copier.GetOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), Excludes:[]string{"dev", "proc", "sys"}, ExpandArchives:false, ChownDirs:(*idtools.IDPair)(0xc00030e9c0), ChmodDirs:(*os.FileMode)(nil), ChownFiles:(*idtools.IDPair)(0xc00030e9d0), ChmodFiles:(*os.FileMode)(nil), StripSetuidBit:false, StripSetgidBit:false, StripStickyBit:false, StripXattrs:false, KeepDirectoryNames:false, Rename:map[string]string(nil), NoDerefSymlinks:false, IgnoreUnreadable:false}, PutOptions:copier.PutOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), DefaultDirOwner:(*idtools.IDPair)(nil), DefaultDirMode:(*os.FileMode)(nil), ChownDirs:(*idtools.IDPair)(nil), ChmodDirs:(*os.FileMode)(nil), ChownFiles:(*idtools.IDPair)(nil), ChmodFiles:(*os.FileMode)(nil), StripXattrs:false, IgnoreXattrErrors:false, IgnoreDevices:false, NoOverwriteDirNonDir:false, Rename:map[string]string(nil)}, MkdirOptions:copier.MkdirOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), ChownNew:(*idtools.IDPair)(nil), ChmodNew:(*os.FileMode)(nil)}}: copier: get: "/"("/"): error copying /extensions/okd/perl-PathTools-3.78-458.fc33.x86_64.rpm: write bulk-writer: broken pipe
Apr 26 14:04:54 bootstrap release-image-download.sh[1856]: W0426 14:04:54.881595    1856 run.go:44] nice failed: running nice -- ionice -c 3 podman cp b0626b6ce1f9915d6558f4387a69c21c3044dacc5a4601aa796bc139aef390db:/ /run/mco-machine-os-content/os-content-104617335 failed: Error: 2 errors occurred:
Apr 26 14:04:54 bootstrap release-image-download.sh[1856]:         * error copying to host: error during bulk transfer for copier.request{Request:"PUT", Root:"/", preservedRoot:"/run/mco-machine-os-content/os-content-104617335", rootPrefix:"/run/mco-machine-os-content/os-content-104617335", Directory:"/", preservedDirectory:"/run/mco-machine-os-content/os-content-104617335", Globs:[]string{}, preservedGlobs:[]string{}, StatOptions:copier.StatOptions{CheckForArchives:false, Excludes:[]string(nil)}, GetOptions:copier.GetOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), Excludes:[]string(nil), ExpandArchives:false, ChownDirs:(*idtools.IDPair)(nil), ChmodDirs:(*os.FileMode)(nil), ChownFiles:(*idtools.IDPair)(nil), ChmodFiles:(*os.FileMode)(nil), StripSetuidBit:false, StripSetgidBit:false, StripStickyBit:false, StripXattrs:false, KeepDirectoryNames:false, Rename:map[string]string(nil), NoDerefSymlinks:false, IgnoreUnreadable:false}, PutOptions:copier.PutOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), DefaultDirOwner:(*idtools.IDPair)(nil), DefaultDirMode:(*os.FileMode)(nil), ChownDirs:(*idtools.IDPair)(0xc0004994c0), ChmodDirs:(*os.FileMode)(nil), ChownFiles:(*idtools.IDPair)(0xc0004994d0), ChmodFiles:(*os.FileMode)(nil), StripXattrs:false, IgnoreXattrErrors:false, IgnoreDevices:true, NoOverwriteDirNonDir:false, Rename:map[string]string(nil)}, MkdirOptions:copier.MkdirOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), ChownNew:(*idtools.IDPair)(nil), ChmodNew:(*os.FileMode)(nil)}}: copier: put: error setting extended attributes on "/extensions/okd/perl-Errno-1.30-468.fc33_1.30-469.fc33.x86_64.drpm": error setting value of extended attribute "user.Zif.MdChecksum[1617881158]" on "/extensions/okd/perl-Errno-1.30-468.fc33_1.30-469.fc33.x86_64.drpm": operation not supported
Apr 26 14:04:54 bootstrap release-image-download.sh[1856]:         * error copying from container: error during bulk transfer for copier.request{Request:"GET", Root:"/", preservedRoot:"/var/lib/containers/storage/overlay/22179495c990ceb5ecf49b26e1eddb6b7fd92c648a3ef9ba831688da4447c628/merged", rootPrefix:"/var/lib/containers/storage/overlay/22179495c990ceb5ecf49b26e1eddb6b7fd92c648a3ef9ba831688da4447c628/merged", Directory:"/", preservedDirectory:"/var/lib/containers/storage/overlay/22179495c990ceb5ecf49b26e1eddb6b7fd92c648a3ef9ba831688da4447c628/merged", Globs:[]string{"/"}, preservedGlobs:[]string{"/var/lib/containers/storage/overlay/22179495c990ceb5ecf49b26e1eddb6b7fd92c648a3ef9ba831688da4447c628/merged/."}, StatOptions:copier.StatOptions{CheckForArchives:false, Excludes:[]string(nil)}, GetOptions:copier.GetOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), Excludes:[]string{"dev", "proc", "sys"}, ExpandArchives:false, ChownDirs:(*idtools.IDPair)(0xc00030e9c0), ChmodDirs:(*os.FileMode)(nil), ChownFiles:(*idtools.IDPair)(0xc00030e9d0), ChmodFiles:(*os.FileMode)(nil), StripSetuidBit:false, StripSetgidBit:false, StripStickyBit:false, StripXattrs:false, KeepDirectoryNames:false, Rename:map[string]string(nil), NoDerefSymlinks:false, IgnoreUnreadable:false}, PutOptions:copier.PutOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), DefaultDirOwner:(*idtools.IDPair)(nil), DefaultDirMode:(*os.FileMode)(nil), ChownDirs:(*idtools.IDPair)(nil), ChmodDirs:(*os.FileMode)(nil), ChownFiles:(*idtools.IDPair)(nil), ChmodFiles:(*os.FileMode)(nil), StripXattrs:false, IgnoreXattrErrors:false, IgnoreDevices:false, NoOverwriteDirNonDir:false, Rename:map[string]string(nil)}, MkdirOptions:copier.MkdirOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), ChownNew:(*idtools.IDPair)(nil), ChmodNew:(*os.FileMode)(nil)}}: copier: get: "/"("/"): error copying /extensions/okd/perl-PathTools-3.78-458.fc33.x86_64.rpm: write bulk-writer: broken pipe
Apr 26 14:04:54 bootstrap release-image-download.sh[1856]: : exit status 125; retrying...
Apr 26 14:06:14 bootstrap release-image-download.sh[1856]: I0426 14:06:14.881867    1856 run.go:18] Running: nice -- ionice -c 3 podman cp b0626b6ce1f9915d6558f4387a69c21c3044dacc5a4601aa796bc139aef390db:/ /run/mco-machine-os-content/os-content-104617335
Apr 26 14:06:15 bootstrap release-image-download.sh[1856]: Error: 2 errors occurred:
Apr 26 14:06:15 bootstrap release-image-download.sh[1856]:         * error copying to host: error during bulk transfer for copier.request{Request:"PUT", Root:"/", preservedRoot:"/run/mco-machine-os-content/os-content-104617335", rootPrefix:"/run/mco-machine-os-content/os-content-104617335", Directory:"/", preservedDirectory:"/run/mco-machine-os-content/os-content-104617335", Globs:[]string{}, preservedGlobs:[]string{}, StatOptions:copier.StatOptions{CheckForArchives:false, Excludes:[]string(nil)}, GetOptions:copier.GetOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), Excludes:[]string(nil), ExpandArchives:false, ChownDirs:(*idtools.IDPair)(nil), ChmodDirs:(*os.FileMode)(nil), ChownFiles:(*idtools.IDPair)(nil), ChmodFiles:(*os.FileMode)(nil), StripSetuidBit:false, StripSetgidBit:false, StripStickyBit:false, StripXattrs:false, KeepDirectoryNames:false, Rename:map[string]string(nil), NoDerefSymlinks:false, IgnoreUnreadable:false}, PutOptions:copier.PutOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), DefaultDirOwner:(*idtools.IDPair)(nil), DefaultDirMode:(*os.FileMode)(nil), ChownDirs:(*idtools.IDPair)(0xc000417b70), ChmodDirs:(*os.FileMode)(nil), ChownFiles:(*idtools.IDPair)(0xc000417be0), ChmodFiles:(*os.FileMode)(nil), StripXattrs:false, IgnoreXattrErrors:false, IgnoreDevices:true, NoOverwriteDirNonDir:false, Rename:map[string]string(nil)}, MkdirOptions:copier.MkdirOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), ChownNew:(*idtools.IDPair)(nil), ChmodNew:(*os.FileMode)(nil)}}: copier: put: error setting extended attributes on "/extensions/okd/perl-Errno-1.30-468.fc33_1.30-469.fc33.x86_64.drpm": error setting value of extended attribute "user.Zif.MdChecksum[1617881158]" on "/extensions/okd/perl-Errno-1.30-468.fc33_1.30-469.fc33.x86_64.drpm": operation not supported
Apr 26 14:06:15 bootstrap release-image-download.sh[1856]:         * error copying from container: error during bulk transfer for copier.request{Request:"GET", Root:"/", preservedRoot:"/var/lib/containers/storage/overlay/22179495c990ceb5ecf49b26e1eddb6b7fd92c648a3ef9ba831688da4447c628/merged", rootPrefix:"/var/lib/containers/storage/overlay/22179495c990ceb5ecf49b26e1eddb6b7fd92c648a3ef9ba831688da4447c628/merged", Directory:"/", preservedDirectory:"/var/lib/containers/storage/overlay/22179495c990ceb5ecf49b26e1eddb6b7fd92c648a3ef9ba831688da4447c628/merged", Globs:[]string{"/"}, preservedGlobs:[]string{"/var/lib/containers/storage/overlay/22179495c990ceb5ecf49b26e1eddb6b7fd92c648a3ef9ba831688da4447c628/merged/."}, StatOptions:copier.StatOptions{CheckForArchives:false, Excludes:[]string(nil)}, GetOptions:copier.GetOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), Excludes:[]string{"dev", "proc", "sys"}, ExpandArchives:false, ChownDirs:(*idtools.IDPair)(0xc00030e900), ChmodDirs:(*os.FileMode)(nil), ChownFiles:(*idtools.IDPair)(0xc00030e910), ChmodFiles:(*os.FileMode)(nil), StripSetuidBit:false, StripSetgidBit:false, StripStickyBit:false, StripXattrs:false, KeepDirectoryNames:false, Rename:map[string]string(nil), NoDerefSymlinks:false, IgnoreUnreadable:false}, PutOptions:copier.PutOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), DefaultDirOwner:(*idtools.IDPair)(nil), DefaultDirMode:(*os.FileMode)(nil), ChownDirs:(*idtools.IDPair)(nil), ChmodDirs:(*os.FileMode)(nil), ChownFiles:(*idtools.IDPair)(nil), ChmodFiles:(*os.FileMode)(nil), StripXattrs:false, IgnoreXattrErrors:false, IgnoreDevices:false, NoOverwriteDirNonDir:false, Rename:map[string]string(nil)}, MkdirOptions:copier.MkdirOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), ChownNew:(*idtools.IDPair)(nil), ChmodNew:(*os.FileMode)(nil)}}: copier: get: "/"("/"): error copying /extensions/okd/perl-NDBM_File-1.15-468.fc33_1.15-469.fc33.x86_64.drpm: write bulk-writer: broken pipe
Apr 26 14:06:15 bootstrap release-image-download.sh[1856]: W0426 14:06:15.864606    1856 run.go:44] nice failed: running nice -- ionice -c 3 podman cp b0626b6ce1f9915d6558f4387a69c21c3044dacc5a4601aa796bc139aef390db:/ /run/mco-machine-os-content/os-content-104617335 failed: Error: 2 errors occurred:
Apr 26 14:06:15 bootstrap release-image-download.sh[1856]:         * error copying to host: error during bulk transfer for copier.request{Request:"PUT", Root:"/", preservedRoot:"/run/mco-machine-os-content/os-content-104617335", rootPrefix:"/run/mco-machine-os-content/os-content-104617335", Directory:"/", preservedDirectory:"/run/mco-machine-os-content/os-content-104617335", Globs:[]string{}, preservedGlobs:[]string{}, StatOptions:copier.StatOptions{CheckForArchives:false, Excludes:[]string(nil)}, GetOptions:copier.GetOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), Excludes:[]string(nil), ExpandArchives:false, ChownDirs:(*idtools.IDPair)(nil), ChmodDirs:(*os.FileMode)(nil), ChownFiles:(*idtools.IDPair)(nil), ChmodFiles:(*os.FileMode)(nil), StripSetuidBit:false, StripSetgidBit:false, StripStickyBit:false, StripXattrs:false, KeepDirectoryNames:false, Rename:map[string]string(nil), NoDerefSymlinks:false, IgnoreUnreadable:false}, PutOptions:copier.PutOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), DefaultDirOwner:(*idtools.IDPair)(nil), DefaultDirMode:(*os.FileMode)(nil), ChownDirs:(*idtools.IDPair)(0xc000417b70), ChmodDirs:(*os.FileMode)(nil), ChownFiles:(*idtools.IDPair)(0xc000417be0), ChmodFiles:(*os.FileMode)(nil), StripXattrs:false, IgnoreXattrErrors:false, IgnoreDevices:true, NoOverwriteDirNonDir:false, Rename:map[string]string(nil)}, MkdirOptions:copier.MkdirOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), ChownNew:(*idtools.IDPair)(nil), ChmodNew:(*os.FileMode)(nil)}}: copier: put: error setting extended attributes on "/extensions/okd/perl-Errno-1.30-468.fc33_1.30-469.fc33.x86_64.drpm": error setting value of extended attribute "user.Zif.MdChecksum[1617881158]" on "/extensions/okd/perl-Errno-1.30-468.fc33_1.30-469.fc33.x86_64.drpm": operation not supported
Apr 26 14:06:15 bootstrap release-image-download.sh[1856]:         * error copying from container: error during bulk transfer for copier.request{Request:"GET", Root:"/", preservedRoot:"/var/lib/containers/storage/overlay/22179495c990ceb5ecf49b26e1eddb6b7fd92c648a3ef9ba831688da4447c628/merged", rootPrefix:"/var/lib/containers/storage/overlay/22179495c990ceb5ecf49b26e1eddb6b7fd92c648a3ef9ba831688da4447c628/merged", Directory:"/", preservedDirectory:"/var/lib/containers/storage/overlay/22179495c990ceb5ecf49b26e1eddb6b7fd92c648a3ef9ba831688da4447c628/merged", Globs:[]string{"/"}, preservedGlobs:[]string{"/var/lib/containers/storage/overlay/22179495c990ceb5ecf49b26e1eddb6b7fd92c648a3ef9ba831688da4447c628/merged/."}, StatOptions:copier.StatOptions{CheckForArchives:false, Excludes:[]string(nil)}, GetOptions:copier.GetOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), Excludes:[]string{"dev", "proc", "sys"}, ExpandArchives:false, ChownDirs:(*idtools.IDPair)(0xc00030e900), ChmodDirs:(*os.FileMode)(nil), ChownFiles:(*idtools.IDPair)(0xc00030e910), ChmodFiles:(*os.FileMode)(nil), StripSetuidBit:false, StripSetgidBit:false, StripStickyBit:false, StripXattrs:false, KeepDirectoryNames:false, Rename:map[string]string(nil), NoDerefSymlinks:false, IgnoreUnreadable:false}, PutOptions:copier.PutOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), DefaultDirOwner:(*idtools.IDPair)(nil), DefaultDirMode:(*os.FileMode)(nil), ChownDirs:(*idtools.IDPair)(nil), ChmodDirs:(*os.FileMode)(nil), ChownFiles:(*idtools.IDPair)(nil), ChmodFiles:(*os.FileMode)(nil), StripXattrs:false, IgnoreXattrErrors:false, IgnoreDevices:false, NoOverwriteDirNonDir:false, Rename:map[string]string(nil)}, MkdirOptions:copier.MkdirOptions{UIDMap:[]idtools.IDMap(nil), GIDMap:[]idtools.IDMap(nil), ChownNew:(*idtools.IDPair)(nil), ChmodNew:(*os.FileMode)(nil)}}: copier: get: "/"("/"): error copying /extensions/okd/perl-NDBM_File-1.15-468.fc33_1.15-469.fc33.x86_64.drpm: write bulk-writer: broken pipe
Apr 26 14:06:15 bootstrap release-image-download.sh[1856]: : exit status 125; retrying...
Apr 26 14:06:15 bootstrap release-image-download.sh[1856]: I0426 14:06:15.865069    1856 rpm-ostree.go:261] Running captured: chcon -R -t var_run_t /run/mco-machine-os-content/os-content-104617335
Apr 26 14:06:16 bootstrap podman[2425]: 2021-04-26 14:06:16.278196505 +0000 UTC m=+0.195799057 container remove b0626b6ce1f9915d6558f4387a69c21c3044dacc5a4601aa796bc139aef390db (image=quay.io/openshift/okd-content@sha256:16da407404e6cedf64b0b6680ddac2bc2e2a3cc6b761fb8ff92d23ef55c7cfdb, name=ostree-container-pivot-ec6f7620-a454-4116-a65a-38464e068573, version=34.20210406.10.0, io.openshift.build.commit.id=570737f3da52045a7082d24effea2f95272524ef, id-machine-config-operator-rpms=sha256:3d734a0fc058388e8bdc244c9b96ff22761828532e9bdee71243a1577d9fcd7d, io.openshift.build.version-display-names=machine-os=Fedora CoreOS, id-artifacts=sha256:291c1695e7c02d77695c75b6ff7999eb35065bcb87b1eac20a680a4f5481b126, io.openshift.build.source-location=https://github.com/openshift/okd-machine-os, io.openshift.build.commit.date=Mon Mar 29 15:36:42 2021 +0000, io.openshift.build.namespace=ci-op-3b4279f7, io.openshift.release.operator=true, io.openshift.build.versions=machine-os=34.20210406.10, io.openshift.build.commit.author=OpenShift Merge Robot <openshift-merge-robot@users.noreply.github.com>, io.buildah.version=1.16.4, io.openshift.build.name=machine-os-content, io.openshift.build.commit.message=Merge pull request #112 from vrutkovs/4.7-no-xattrs, io.openshift.build.commit.ref=release-4.7)
Apr 26 14:06:16 bootstrap release-image-download.sh[1856]: I0426 14:06:16.284552    1856 rpm-ostree.go:261] Running captured: rpm-ostree status --json
Apr 26 14:06:19 bootstrap release-image-download.sh[1856]: I0426 14:06:19.243865    1856 rpm-ostree.go:184] Current origin is not custom
Apr 26 14:06:21 bootstrap release-image-download.sh[1856]: I0426 14:06:21.232054    1856 rpm-ostree.go:216] No com.coreos.ostree-commit label found in metadata! Inspecting...
Apr 26 14:06:21 bootstrap release-image-download.sh[1856]: I0426 14:06:21.233132    1856 rpm-ostree.go:261] Running captured: ostree refs --repo /run/mco-machine-os-content/os-content-104617335/srv/repo
Apr 26 14:06:23 bootstrap podman[2938]: 2021-04-26 14:06:23.224761493 +0000 UTC m=+1.860223334 image remove 6452c626a971168f1a0abe2a86c0f65665288be0d6fa82ef713e8a960bf4cc3c quay.io/openshift/okd-content@sha256:16da407404e6cedf64b0b6680ddac2bc2e2a3cc6b761fb8ff92d23ef55c7cfdb
Apr 26 14:06:23 bootstrap release-image-download.sh[1856]: error: error running ostree refs --repo /run/mco-machine-os-content/os-content-104617335/srv/repo: error: opening repo: opendir(/run/mco-machine-os-content/os-content-104617335/srv/repo): No such file or directory
Apr 26 14:06:23 bootstrap release-image-download.sh[1856]: : exit status 1
Apr 26 14:06:23 bootstrap systemd[1]: release-image.service: Main process exited, code=exited, status=1/FAILURE
Apr 26 14:06:23 bootstrap systemd[1]: release-image.service: Failed with result 'exit-code'.
Apr 26 14:06:23 bootstrap systemd[1]: Failed to start Download the OpenShift Release Image.
Apr 26 14:06:23 bootstrap systemd[1]: Dependency failed for Bootstrap a Kubernetes cluster.
Apr 26 14:06:23 bootstrap systemd[1]: bootkube.service: Job bootkube.service/start failed with result 'dependency'.
Apr 26 14:06:34 bootstrap systemd[1]: Starting Download the OpenShift Release Image...
Apr 26 14:06:34 bootstrap release-image-download.sh[3069]: Pulling quay.io/openshift/okd@sha256:4414db9e8945504d1d5423ec11d64df9fa8b4aa60d978f8b4bdb3d0720800741...
Apr 26 14:06:36 bootstrap podman[3070]: 2021-04-26 14:06:36.579698566 +0000 UTC m=+2.293567717 image pull  
Apr 26 14:06:36 bootstrap release-image-download.sh[3070]: f388ece1e5537b735eb48b61164eb15a9784f8192c4637b51ceb01b7971505fa
Apr 26 14:06:36 bootstrap systemd[1]: Finished Download the OpenShift Release Image.
'/home/serinf/Untitled Document 1'

**Comment by vrutkovs** [comment]
Looks a lot like #566, are you sure the starting image is 33.20210217.3.0? Please attach a log bundle

**Comment by EmanueleMiz** [comment]
Thanks a lot, it gives me another error, do you have any solution?

Apr 27 22:14:28 bootstrap bootkube.sh[32273]: [#4889] failed to fetch discovery: Get "https://localhost:6443/api?timeout=32s": x509: certificate has expired or is not yet valid: current time 2021-04-27T22:14:28Z is after 2021-04-22T11:43:54Z

This is bootstrap node.

**Comment by cgruver** [comment]
Make sure your system clocks are correct on the hypervisor machines.  Remember that the Bootstrap node will want to set itself to UTC.

**Comment by fedorax** [comment]
@EmanueleMiz
The same problem happened to me.

Maybe when you run the create manifest command, you are reusing the previous folder?
There are hidden files ".openshift_install.log" and ".openshift_install_state.json" in the previous folder.

`rm ./bare-metal/.* && openshift-install create manifests --dir=./bare-metal`

I have had success by either deleting these hidden files or recreating the folder and running the create manifest command.

**Comment by EmanueleMiz** [comment]
I create a new machine to install the OKD,
Using fedora version 33.20210217.3.0 but but only master machine start run
The worker machine not start.
It would appear that the kubernetes api doesn't work.
i see seme error "https://localhost:6443/api?timeout=32s": x509: certificate has expired or is not yet valid" on the bootstrap  machine 
The bundle log is not generated

**Comment by vrutkovs** [comment]
>The bundle log is not generated

https://docs.okd.io/latest/installing/installing-troubleshooting.html#installation-manually-gathering-logs-with-SSH_installing-troubleshooting
https://github.com/openshift/installer/blob/master/docs/user/troubleshootingbootstrap.md

**Comment by EmanueleMiz** [comment]
this is fine ?
[okd log](https://github.com/openshift/okd/files/6590473/okd.log)

**Comment by vrutkovs** [comment]
>current time 2021-06-03T09:14:51Z is after 2021-05-22T15:10:22Z

Your bootstrap is valid for 24hrs only. Seems you need to regenerate the ignition files and restart the installation

**Comment by EmanueleMiz** [comment]
i restart the installation but it gave same problems

[log-bundle-20210603151201.tar.gz](https://github.com/openshift/okd/files/6591868/log-bundle-20210603151201.tar.gz)

**Comment by EmanueleMiz** [comment]
[log-bundle-20210603164422.tar.gz](https://github.com/openshift/okd/files/6592453/log-bundle-20210603164422.tar.gz)
whit all master machines

**Comment by vrutkovs** [comment]
Your bootstrap has completed though:
>Jun 03 14:23:07 bootstrap bootkube.sh[58992]: Sending bootstrap-finished event.Tearing down temporary bootstrap control plane...

Note that there are a lot of
>2021-06-03 14:23:45.656681 W | etcdserver: read-only range request "key:\"/kubernetes.io/configmaps/openshift-apiserver/config\" " with result "range_response_count:1 size:1270" took too long (128.077697ms) to execute

in etcd log. Your backing storage is apparently not fast enough for etcd. See [this article](https://www.ibm.com/cloud/blog/using-fio-to-tell-whether-your-storage-is-fast-enough-for-etcd) to find out more

**Comment by EmanueleMiz** [comment]
It seems strange to me it is a storage connected in fiber on the vSphere cluster the access times are less than 1m
And anyway, the installation of the master machines starts immediately but the workers one doesn't start
all machines have 4 vCPU and 16 Gb Ram
 Where am I doing wrong ?
